lib/ASTGen/CMakeLists.txt

```diff
@@ -96,6 +96,7 @@ endforeach()
 
 # Prior to 5.9, we have to use the experimental flag for C++ interop.
 if (CMAKE_Swift_COMPILER_VERSION VERSION_LESS 5.9)
+  set(cxx_interop_flag_prefix "-Xfrontend ")
```

Comment:
SwiftCompilerSources has the same problem. I think you're better off just changing this to:
```
if (CMAKE_Swift_COMPILER_VERSION VERSION_LESS 5.9)
  list(APPEND compile_options "-Xfrontend" "-enable-experimental-cxx-interop")
else()
  list(APPEND compile_options "-cxx-interoperability-mode=default")
  set(cxx_interop_flag "-cxx-interoperability-mode=default")
endif()
```

(Note it's `swift_compile_options` in SwiftCompilerSources)